catalog/imaging/imaging-transformer-nitf/src/main/java/org/codice/alliance/transformer/nitf/NitfUtilities.java

```diff
@@ -0,0 +1,77 @@
+/**
+ * Copyright (c) Codice Foundation
+ *
+ * <p>This is free software: you can redistribute it and/or modify it under the terms of the GNU
+ * Lesser General Public License as published by the Free Software Foundation, either version 3 of
+ * the License, or any later version.
+ *
+ * <p>This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY;
+ * without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Lesser General Public License for more details. A copy of the GNU Lesser General Public
+ * License is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ */
+package org.codice.alliance.transformer.nitf;
+
+import java.time.Instant;
+import java.time.ZonedDateTime;
+import java.util.Date;
+import java.util.List;
+import javax.annotation.Nullable;
+import org.apache.commons.collections.CollectionUtils;
+import org.codice.ddf.internal.country.converter.api.CountryCodeConverter;
+import org.codice.imaging.nitf.core.common.DateTime;
+
+/** General NITF utility functions */
+public class NitfUtilities {
```

Comment:
❓ Should we make the name a little bit more specific to prevent this class from becoming a dumping ground for NITF utility methods, e.g., `NiftHeaderFieldConverters` or something similar? If we do, we could consider removing the `convert` prefix from the method names.